### PR TITLE
added solution to hw-3.1

### PR DESCRIPTION
**Pull request by SpringWay**

### CODE REVIEW

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 1]
```diff
@@ -7,7 +7,11 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double number) throws IllegalArgumentException {
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/SqrtOperation.java]: remove empty line

Comment (SpringWay) [reply src/main/java/core/basesyntax/SqrtOperation.java]: done:)

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 2]
```diff
@@ -7,7 +7,10 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double number) throws IllegalArgumentException {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/SqrtOperation.java]: can we remove throws IllegalArgumentException ?

Comment (SpringWay) [reply src/main/java/core/basesyntax/SqrtOperation.java]: yes

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**